server/etcdmain/etcd.go

```diff
@@ -90,6 +90,10 @@ func startEtcdOrProxyV2(args []string) {
 		lg.Info("failed to detect default host", zap.Error(dhErr))
 	}
 
+	if cfg.ec.Name == embed.DefaultName {
+		lg.Warn("the server is using default name, which might cause error that failed the startup.")
+	}
```

Comment:
```suggestion
	// If `--name` isn't configured, then multiple members may have the same "default" name. 
	// When adding a new member with the "default" name as well, etcd may regards its peerURL 
	// as one additional peerURL of the existing member which has the same "default" name, 
	// because each member can have multiple client or peer URLs. 
	// Please refer to https://github.com/etcd-io/etcd/issues/13757
	if cfg.ec.Name == embed.DefaultName {
		lg.Warn("the server is using default name, which might cause error that failed the startup.")
	}
```